### DISCUSSION TITLE
Cookies in the uniapp mini program

### DISCUSSION THREAD

**Discussion by Lin-Van** [alovajs/alova#70] [Q&A] [answered]
Cookies cannot be automatically carried in the uniapp mini program. 
Is there a solution?
Is it necessary to manually set it to the header.

**Comment by JOU-amjs** [comment, marked as answer]
小程序是不支持cookie的，你需要自己处理，可以看这边参考以下 https://blog.csdn.net/Ghjkku/article/details/125950177

如果是h5的话，需要在请求中打开`withCredentials`
![image](https://github.com/alovajs/alova/assets/29848971/096ec393-949e-4d84-aceb-8998126cfb07)

**Reply by Lin-Van** [reply]
好的，谢谢，我现在大概也是这么处理，在createAlova时的beforeRequest，封装到method.config.headers中。
